app/viewmodels/authorisedOfficials/AddedOneAuthorisedOfficialHelper.scala

```diff
@@ -34,158 +29,48 @@ class AddedOneAuthorisedOfficialHelper(index: Index) (override val userAnswers:
-    }.fold(List[SummaryListRow]())(_.toList)
+  def authOfficialNamesRows: Option[SummaryListRow] =
+    answerFullame(AuthorisedOfficialsNamePage(index), authOfficialRoutes.AuthorisedOfficialsNameController.onPageLoad(CheckMode, index))
 
-  def authorisedOfficialsPhoneNumberRow: List[SummaryListRow] =
-    userAnswers.get(AuthorisedOfficialsPhoneNumberPage(index)).map{ contact =>
-      answerOfficialsPhoneNumber(contact, authOfficialRoutes.AuthorisedOfficialsPhoneNumberController.onPageLoad(CheckMode, index))
-    }.fold(List[SummaryListRow]())(_.toList)
+  def authOfficialDobRows: Option[SummaryListRow] =
+    answer(AuthorisedOfficialsDOBPage(index), authOfficialRoutes.AuthorisedOfficialsDOBController.onPageLoad(CheckMode, index))
 
-  def authOfficialPositionRows: List[SummaryListRow] =
-    userAnswers.get(AuthorisedOfficialsPositionPage(index)).map{ position =>
-      answerAuthOfficialPosition(position, authOfficialRoutes.AuthorisedOfficialsPositionController.onPageLoad(CheckMode, index))
-    }.fold(List[SummaryListRow]())(_.toList)
+  def authOfficialMainPhoneNoRows: Option[SummaryListRow] =
+    answerMainPhoneNo(AuthorisedOfficialsPhoneNumberPage(index),
+      authOfficialRoutes.AuthorisedOfficialsPhoneNumberController.onPageLoad(CheckMode, index),
+      messagePrefix = "authorisedOfficialsPhoneNumber.mainPhoneNumber")
 
-  def authOfficialHasNINORow: List[SummaryListRow] =
-    userAnswers.get(IsAuthorisedOfficialNinoPage(index)).map{ boolean =>
-      answerAuthOfficialHasNINO(boolean, authOfficialRoutes.IsAuthorisedOfficialNinoController.onPageLoad(CheckMode, index))
-    }.fold(List[SummaryListRow]())(_.toList)
+  def authOfficialAlternativePhoneNoRows: Option[SummaryListRow] =
+    answerAlternativePhoneNo(AuthorisedOfficialsPhoneNumberPage(index),
+      authOfficialRoutes.AuthorisedOfficialsPhoneNumberController.onPageLoad(CheckMode, index),
+      messagePrefix = "authorisedOfficialsPhoneNumber.alternativePhoneNumber")
 
-  def authOfficialNINoRows: List[SummaryListRow] =
-    userAnswers.get(AuthorisedOfficialsNinoPage(index)).map{ NINO =>
-      answerAuthOfficialsNINO(NINO, authOfficialRoutes.AuthorisedOfficialsNinoController.onPageLoad(CheckMode, index))
-    }.fold(List[SummaryListRow]())(_.toList)
+  def authOfficialPositionRows: Option[SummaryListRow] =
+    answerPrefix(AuthorisedOfficialsPositionPage(index),
+      authOfficialRoutes.AuthorisedOfficialsPositionController.onPageLoad(CheckMode, index),
+      answerIsMsgKey = true,
+      messagePrefix = "officialsPosition")
 
-  def authOfficialAddressRow: List[SummaryListRow] =
-    userAnswers.get(AuthorisedOfficialAddressLookupPage(index)).map{ address =>
-      answerAuthOfficialAddress(address, controllers.addressLookup.routes.AuthorisedOfficialsAddressLookupController.initializeJourney(index))
-    }.fold(List[SummaryListRow]())(_.toList)
+  def authOfficialHasNINORow: Option[SummaryListRow] = {
```

Comment:
method Name can we change it to otherOfficialHasNinoRow simillarly otherOfficialNinoRows